### PR TITLE
chore: update default copy for FindProperty plot a new address 

### PR DESCRIPTION
**Pull request by jessicamcinchak**
Per Doncaster feedback, see Trello ticket

New components only, not migrating existing content in this case

### CODE REVIEW

**Comment by jessicamcinchak** [comment]
[Update default copy of Find Property 'plot new addresses without a UPRN'](https://trello.com/c/bdDppPEN/2800-update-default-copy-of-find-property-plot-new-addresses-without-a-uprn)

File: editor.planx.uk/src/@planx/components/FindProperty/model.ts [hunk 1]
```diff
@@ -54,5 +56,8 @@ export interface SiteAddress extends MinimumSiteAddress {
 }
 
 export const DEFAULT_TITLE = "Find the property" as const;
-export const DEFAULT_NEW_ADDRESS_TITLE = "Propose a new address" as const;
+export const DEFAULT_NEW_ADDRESS_TITLE =
+  "Click or tap at where the property is on the map and name it below" as const;
+export const DEFAULT_NEW_ADDRESS_DESCRIPTION =
+  "You will need to select a location and provide a name to continue" as const;
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/@planx/components/FindProperty/model.ts:62]: These form values also appear to be referenced in `planx-new/editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx` which is actually redundant I think.

We should be able to mark these properties as no longer optional in the `FindProperty` interface and tidy this up a bit.

**Review by DafyddLlyr — APPROVED** [review]
Perfect, nice and simple!

File: editor.planx.uk/src/@planx/components/FindProperty/model.ts (hunk 1 above)
Comment (jessicamcinchak) [reply editor.planx.uk/src/@planx/components/FindProperty/model.ts:62]: Yep good shout - cleaned up from index & Editor placeholders now !

**Pull request merged**